site/src/sphinx/zookeeper.rst

```diff
@@ -0,0 +1,55 @@
+.. _`Apache ZooKeeper`: https://zookeeper.apache.org/
+.. _`com.linecorp.armeria.client.zookeeper`: apidocs/index.html?com/linecorp/armeria/client/zookeeper/package-summary.html
+.. _`com.linecorp.armeria.server.zookeeper`: apidocs/index.html?com/linecorp/armeria/server/zookeeper/package-summary.html
+.. _`StoreType`: apidocs/index.html?com/linecorp/armeria/client/zookeeper/StoreType.html
+
+Automatic service discovery with ZooKeeper
+===========================================
+You can put your Endpoints information into an zNode in `Apache ZooKeeper`_ Cluster, as a node value or as the child value of a node, like the following:
```

Comment:
- an zNode -> a zNode
- Apache ZooKeeper Cluster -> an Apache ZooKeeper cluster (add 'an' and C -> c)
- as the child value -> as the child values